source/includes/api/_metadata.md

```diff
@@ -1,188 +1,436 @@
 # Metadata
 
-## Metadata definition
+## TODO LIST
 
-Metadata is data concerning another resource available on this API. It only exists when associated with another resource - currently supported resource types are datasets, widgets or layers. Note that, although there might be several metadata per resource, each resource will have - at most - one metadata element per `application` and `language` pair. As such, the `application` and `language` attributes are required and it is mandatory to include them in when creating/updating metadata.
++ check if a vocab is used for language, and application
```

Comment:
Didn't get this TODO, what was the question here?

Reply:
How does a user know the valid values of `language` and `application`? Is there a `enum` definition somewhere or can they consult a resource that describes the valid values?